### DISCUSSION TITLE
Multiple Instances of Same Docket Returned?

### DISCUSSION THREAD

**Discussion by mathchou** [freelawproject/courtlistener#3704] [Q&A] [answered]
Hi All,

I'm trying to find a list of ALL docket entries for a specific case. 

 3:15-cr-00700-FAB
Case title: USA v. Echevarria-Irizarry et al

I run the following search, ensuring the date_filed__year is 2015 and the docket_number_core is 1500700:

> https://www.courtlistener.com/api/rest/v3/dockets/?court=prd&date_filed__year=2015&docket_number_core=1500700

However, it returns 16 results, all seemingly the same case, but with only partial docket entries in each json in the 

> "filepath_ia_json": "https://archive.org/download/gov.uscourts.prd.122289/gov.uscourts.prd.122289.docket.json"

entry.

I'm a bit confused by this. Can I use the API to find all the most up to date docket entries for this specific case?

Thanks,
Mike

**Comment by mlissner** [comment, marked as answer]
Hi, yes, you can use the API to do this, but we do have an issue with multi-defendant criminal cases, where we wind up making multiple dockets like this. 

The trick to get the docket entries is to use the docket-entry API, not the docket API, and to filter it to a particular docket. 

That API is not open by default, but if you drop us a line with our contact form, we can usually open it up for different purposes.

**Reply by mathchou** [reply]
Wonderful. I've run:

> https://www.courtlistener.com/api/rest/v3/docket-entries/?court=prd&docket__docket_number_core=1300156&order_by=id

for another case I'm interested in, and indeed I get all of the docket entries for the case. Thank you very much!
